habushu-mixology-consumer/pom.xml

```diff
@@ -14,6 +14,7 @@
     <packaging>habushu</packaging>
 
     <build>
```

Comment:
I (Informational): I messed this up and took it out of the prior PR as I thought it was superfluous.  I think the test passed in that case b/c the cached version had this change cached.  So I didn't find it until I forced the final test via a clean cache.